x/wasm/keeper/msg_server.go

```diff
@@ -11,26 +11,14 @@ import (
 
 var _ types.MsgServer = msgServer{}
 
-// abstract type
-type paramStore interface {
-	GetAuthority() string
-	SetParams(ctx sdk.Context, params types.Params) error
-}
-
 // grpc message server implementation
 type msgServer struct {
-	keeper     types.ContractOpsKeeper
-	paramStore paramStore
+	keeper Keeper
```

Comment:
let's use a pointer here, in case we need to mock some functionality later in tests